src/client/app/frameworks/i18n.framework/services/multilingual.service.ts

```diff
@@ -43,39 +43,42 @@ export const multilingualReducer: ActionReducer<MultilingualStateI> = (state: Mu
 // service
 @Injectable()
 export class MultilingualService extends Analytics {
-  
+
   // default supported languages
   // see main.ts bootstrap for example of how to provide different value
   public static SUPPORTED_LANGUAGES: Array<ILang> = [
-    { code: 'en', title: 'English' }
+    {code: 'en', title: 'English'}
   ];
-  
+
+  private static DEFAULT_LANG: string = 'en';
```

Comment:
Funny to see a `private` static variable?
Would be nice to set this during bootstrap for any platform so maybe make `public`